packages/components/pie-icon-button/src/index.ts

```diff
@@ -10,11 +10,11 @@ export { ICON_BUTTON_VARIANT };
 const componentSelector = 'pie-icon-button';
 
 export class PieIconButton extends LitElement {
-    @property()
+    @property({ reflect: true })
```

Comment:
Would be good to remove the `reflect` here in favour of the styling like in `pie-button`